pom.xml

```diff
@@ -202,7 +203,7 @@
             <extension>
                 <groupId>com.vackosar.gitflowincrementalbuilder</groupId>
                 <artifactId>gitflow-incremental-builder</artifactId>
-                <version>3.1</version>
+                <version>3.4</version>
```

Comment:
This is just bumped to match master.

Reply:
@brendan-hofmann also the failOnError argument isn't valid/supported in 3.1 -- it caused the build to fail without upgrading.